index.js

```diff
@@ -203,7 +201,7 @@ async function run(dir) {
 
   log(`\tSetting up UI`);
   log(`\t\tInitializing create-react-app...`);
-  spawn.sync("create-react-app", [uiDirectory]);
+  spawn.sync("npx", ["create-react-app", uiDirectory]);
```

Comment:
not familiar with npx and what's it for. It's not the typo, is it? Just to confirm.

Reply:
Spoke with Samrit about this one. `create-react-app` is deprecated, `npx create-react-app` is the new command